src/saltext/vmware/utils/license_mgr.py

```diff
@@ -0,0 +1,435 @@
+# Copyright 2021 VMware, Inc.
+# SPDX-License-Identifier: Apache-2.0
+# pylint: disable=no-name-in-module
+import logging
+
+# noreorder
+import salt.exceptions
+import salt.modules.cmdmod
+import salt.utils.path
+import salt.utils.platform
+import salt.utils.stringutils
+import saltext.vmware.utils.cluster as utils_cluster
+import saltext.vmware.utils.datacenter as utils_datacenter
+import saltext.vmware.utils.esxi as utils_esxi
+from saltext.vmware.utils.common import get_service_content as get_service_content
+
+try:
+    from pyVmomi import vim, vmodl
+
+    HAS_PYVMOMI = True
+except ImportError:
+    HAS_PYVMOMI = False
+
+log = logging.getLogger(__name__)
+
+
+def _list_lic_keys(licenses):
+    """
+    Return a list of license keys
+
+    licenses:
+       vCenter licenses as made available from licenseManager
+
+    Note:
+        skips evaluation licenses
+    """
+    lic_keys = []
+    for lic in licenses:
+        if lic.used is None:
+            continue
+        lic_keys.append(lic.licenseKey)
+    return lic_keys
+
+
+def _find_lic_for_key(licenses, license_key):
+    """
+    Return license which matches specified license_key
+
+    licenses:
+       vCenter licenses as made available from licenseManager
+
+    license_key
+        license key to search for
+
+    Note:
+        skips evaluation licenses
+    """
+    lic_keys = []
+    for lic in licenses:
+        if lic.used is None:
+            continue
+        if license_key == lic.licenseKey:
+            return lic
+    return None
+
+
+def is_vcenter(service_instance):
+    """
+    Test if service_instance represents vCenter,
+
+    service_instance
+        The Service Instance to check if it represents a vCenter
+
+    Return:
+        True  - vCenter
+        False - not a vCenter
+        None - neither of the above
```

Comment:
**suggestion** just missed the None in the docstring here :slightly_smiling_face:

Reply:
fixed